### DISCUSSION TITLE
copilot not able to reference existing ECS cluster with ec2 launch type + unable to spin up ECs cluster with with ec2 launch type

### DISCUSSION THREAD

**Discussion by pratikm13** [aws/copilot-cli#5839] [General]
Dear Team,

I have situation where my application is live on ECS with fargate tasks with efs storage , but due to bad I/O experience with EFs storage with plan to use Ebs storage with ECs cluster .

So I  spin up a ECS cluster with EC2 launch type using cloudformation 

But when I try use copilot to make use of existing ECS cluster it is not possible and something that should be introduced asap or atleast make copilot to spin up ECS cluster with ec2 launch type possible through copilot  and also support mounting of ebs volumes

**Comment by huanjani** [comment]
https://github.com/aws/copilot-cli/issues/1708#issuecomment-731371906
